### PR TITLE
Removes user profile links on DevHub activity feed

### LINKED ISSUE #9339: Remove user profile links on DevHub activity feed
From https://github.com/mozilla/addons-frontend/issues/6118, there are two places where we're still linking to arbitrary user profile pages. There's the Recent Activity column in `/developers/addons`, and the activity feed in `/developers/feed`. Those usernames should show up without a link.

### PR DESCRIPTION
**Pull request by Abhi-khandelwal**
Fixes #9339

### CODE REVIEW

**Comment by Abhi-khandelwal** [comment]
@diox can you please help me with test case.

File: src/olympia/devhub/templates/devhub/addons/dashboard.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h3>
             {{ item }}
             <span class="activity-timestamp">
             {% trans user=item.user|user_link, ago=item.created|timesince %}
```
Comment (eviljeff) [inline src/olympia/devhub/templates/devhub/addons/dashboard.html]: It's this line you'll want to change, where `user` is set to the result of the `user_link` jinja filter.  To just have the name but not the link we can just use `user=user.name` instead.

File: src/olympia/devhub/templates/devhub/addons/activity.html [hunk 2]
```diff
@@ -34,9 +34,8 @@ <h2>{{ _('Recent Activity for My Add-ons') }}</h2>
                    ago=item.created|timesince,
                    iso=item.created|isotime,
                    pretty=item.created|datetime %}
-          <time datetime="{{ iso }}" title="{{ pretty }}">{{ ago }}</time>
-          by {{ user }}
```
Comment (eviljeff) [inline src/olympia/devhub/templates/devhub/addons/activity.html]: see the comment for dashboard.html

Comment (eviljeff) [inline src/olympia/devhub/templates/devhub/addons/activity.html]: see the comment for dashboard.html

**Comment by Abhi-khandelwal** [comment]
@eviljeff can you please help me with test cases.

File: src/olympia/devhub/templates/devhub/addons/dashboard.html [hunk 3]
```diff
@@ -44,7 +44,7 @@ <h3>
           <li class="item">
             {{ item }}
             <span class="activity-timestamp">
-            {% trans user=item.user|user_link, ago=item.created|timesince %}
+            {% trans user=item.user, ago=item.created|timesince %}
```
Comment (eviljeff) [inline src/olympia/devhub/templates/devhub/addons/dashboard.html]: actually, `item.user|user_link` should become `item.author_name`

**Comment by eviljeff** [comment]
You can include the following in devhub/tests/test_views.py under TestActivityFeed:
```
    def test_addons_dashboard_name(self):
        self.add_log()
        res = self.client.get(reverse('devhub.addons'))
        doc = pq(res.content)
        timestamp = doc('.recent-activity li.item span.activity-timestamp')
        assert len(timestamp) == 1
        assert self.action_user.name
        assert self.action_user.name in timestamp.html()
        assert '<a href=' not in timestamp.html()

    def test_addons_dashboard_reviewer_name(self):
        self.action_user.update(reviewer_name='bob')
        self.add_log(action=amo.LOG.APPROVE_VERSION)
        res = self.client.get(reverse('devhub.addons'))
        doc = pq(res.content)
        timestamp = doc('.recent-activity li.item span.activity-timestamp')
        assert len(timestamp) == 1
        assert self.action_user.name
        assert self.action_user.name not in timestamp.html()
        assert self.action_user.reviewer_name in timestamp.html()
        assert '<a href=' not in timestamp.html()
```

**Comment by caitmuenster** [comment]
Thanks, @Abhi-khandelwal! 🎉 Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition#) and your Mozillians profile has been vouched. 

Welcome aboard! I look forward to seeing you around the project.

**Comment by Abhi-khandelwal** [comment]
Hi Caitlin ,
Thank you so much.

**Pull request merged**